debs/trusty/archivematica/Dockerfile

```diff
@@ -15,3 +15,9 @@ RUN apt-get update && apt-get install -y libffi-dev libssl-dev libxslt-dev pytho
 
 RUN curl -sL https://deb.nodesource.com/setup_8.x |  bash - && apt-get install -y nodejs
 
+# Set Path
+ENV PATH=/usr/local/bin:$PATH
+
+# Update setuptools and virtualenv
+RUN /usr/local/bin/pip install -U setuptools
+RUN /usr/local/bin/pip install -U virtualenv
```

Comment:
Can we do `/usr/local/bin/pip install -U setuptools virtualenv` instead so we avoid the extra layer?